Add Jelly Snippets plugin

rabirabirara opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/rabirabirara/obsidian-jelly-snippets

## Release Checklist
- [ ] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
["fundingUrl": "",](https://github.com/rabirabirara/obsidian-jelly-snippets/blob/7c6ad9b35fdf374d8a90f6da960caff50e8bebe7/manifest.json#L9)
If you don't have one. remove this.

Your plugin will not work in pop out windows.
See here how to update your plugin to support pop out windows:
<https://forum.obsidian.md/t/plugin-developers-how-to-update-your-plugin-to-support-pop-out-windows/40228>

rabirabirara commented:
Okay; I updated my repository with https://github.com/rabirabirara/obsidian-jelly-snippets/commit/06ced0a5a2e1380daf25ae5ec653cdfd43af5dcf.

Removed `fundingUrl` from `manifest.json`.

For popup windows, I registered an event so that whenever a new window opened, the keydown DOM event would be registered again for that active window. I hope this is satisfactory.  Since I am using `register...Event()` I am expecting that memory will take care of itself.

liamcain commented:
- [getActiveViewOfType](https://github.com/rabirabirara/obsidian-jelly-snippets/blob/8ebc66e486c68d8cbe8aa243b3d573672f509d22/main.ts#L74) you should use [`activeEditor`](https://github.com/obsidianmd/obsidian-api/blob/38dd22168d2925086371bfc59e36fd9121527a39/obsidian.d.ts#L4009) instead. getActiveViewOfType won't work if the user is editing a file inside of Canvas.
